web/server/vue-cli/src/components/Report/SelectReviewStatus.vue

```diff
@@ -57,6 +57,33 @@
 
       <v-card-text class="pa-0">
         <v-container>
+          <v-alert
+            v-if="reportsReviewedInSource.length != 0"
+            dense
+            type="error"
+          >
+            <p>
+              Review status of the following reports are given as source code
+              comment. Their review status will not change.
```

Comment:
It would be good to rephrase this sentence to mention that we do not advice to use source code comments and changing the review status on the GUI in the same product.

Also we have to mention if the user still wan't to change the review status, he is able to do it but it will change the review status for future and already existing reports which do not have a source code comment.